test/Plugins.js

```diff
@@ -0,0 +1,16 @@
+import {compile, serialize, pluginMiddleware, prefix} from "../index.js"
+
+describe('prefixer plugin', () => {
+    const prefixPlugin = node => {
+        switch (node.type) {
+            case 'decl':
+                node.value = prefix(node.value, node.props.length)
+        }
+    }
+
+    const stylis = string => serialize(pluginMiddleware([prefixPlugin])(compile(`.user{${string}}`)))
```

Comment:
this a simplistic version of the API, but I dont know if we can do much better if we want to keep existing `serialize` rather combining plugins & serialize in a single thing (maybe an alternative serialize? `serializeWithPlugins`?)